src/ontology/pato-edit.obo

```diff
@@ -21676,6 +21676,13 @@ is_a: PATO:0060001 ! sex-specific
 created_by: http://orcid.org/0000-0002-1373-1705
 creation_date: 2019-08-23T08:51:22Z
 
+[Term]
+id: PATO:0065001
+name: lysed
+def: "the breakdown, disintegration, destruction of an entity due to the breaking of cell membranes by any means." [https://orcid.org/0000-0002-3528-5267]
```

Comment:
First character in def should be capital, and the cross reference should perhaps be to  paper that describes the concept rather than your orcid?